test/integration/MultiIssuanceTokenClass.spec.ts

```diff
@@ -289,7 +289,7 @@ describe('Multi Issuance Token Class', () => {
     
 
     it('should fail minting with the wrong identity', function(done) {
-        this.timeout(50000)
+        this.timeout(500000)
```

Comment:
Why do we need this insane timeout?

Reply:
Hehe, I did that when I was debugging, must have committed it by mistake ...